themes/gatsby-starter-theme/gatsby-config.js

```diff
@@ -3,7 +3,8 @@ module.exports = {
     {
       resolve: `gatsby-theme-notes`,
       options: {
-        mdx: false,
+        mdx: true,
```

Comment:
@ChristopherBiscardi are we still going to need to disable MDX like this for the time being? Wasn't sure what's needed to dedupe the MDX plugins or where we left off with that

Reply:
Yeah, for now we'll have to turn it off with this API since we haven't implemented a way for plugins to merge configs yet (AFAIK).